cluster/common.sh

```diff
@@ -358,6 +367,9 @@ function generate-etcd-cert() {
   local GEN_ETCD_CA_CERT=${GEN_ETCD_CA_CERT:-}
   local GEN_ETCD_CA_KEY=${GEN_ETCD_CA_KEY:-}
 
+  local ca_cert=${ca_cert:-}
+  local ca_key=${ca_key:-}
```

Comment:
why this change?

Reply:
these 2 variable are referenced in this function, but they are defined outside, we should get their value first:
```bash
  if [[ -n "${GEN_ETCD_CA_CERT}" && -n "${GEN_ETCD_CA_KEY}" ]]; then
    echo "${ca_cert}" | base64 --decode | gunzip > ca.pem
    echo "${ca_key}" | base64 --decode > ca-key.pem
  fi
```

Reply:
they're not actually local though, and we probably shouldn't allow unset values

Reply:
we should not redefine these with the same name nor should we allow unset values for this purpose

Reply:
Keep as it was and update comment, please check.